### PR TITLE
NGGW-89 add basic git branch information to webprofiler

### PR DESCRIPTION
**Pull request by iherak**

### CODE REVIEW

File: bundle/DataCollector/GitDataCollector.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+final class GitDataCollector extends DataCollector
+{
+    public function collect(Request $request, Response $response, \Exception $exception = null)
+    {
+        $repository = new Repository();
+
+        $branch = $repository->getInfoOperator()->getCurrentBranch();
```
Comment (emodric) [inline bundle/DataCollector/GitDataCollector.php:18]: Can we add info if the current branch is tracking a remote branch?

For example:

`1.11 (origin/1.11)`

Comment (iherak) [reply bundle/DataCollector/GitDataCollector.php:18]: Not in an easy way (using provided library).
It would require some custom parsing..

Comment (emodric) [reply bundle/DataCollector/GitDataCollector.php:18]: Then merging as is. Good enough :)

**Comment by emodric** [comment]
Thanks @iherak !

**Pull request merged**